.flowconfig

```diff
@@ -20,3 +13,10 @@ untyped-type-import
 
 [version]
 ^0.91.0
+
+[options]
```

Comment:
why moving this to the bottom?

Reply:
so that the `echo` in the lint run inserts `max-worker` under `[options]`

Reply:
seems like it messed up, though 😀

Reply:
Because of the max workers change in the CI

Reply:
Needs to be `>>` tho I think

Reply:
yes, pushed it